Makefile

```diff
@@ -1,3 +1,5 @@
+VERSION = 0.2.0
```

Comment:
Should the `VERSION` be specific to ABI or synchronized with each release?

Reply:
In my original design, it should be synchronized with each release.